CalibCalorimetry/HcalPlugins/src/HcalTextCalibrations.h

```diff
@@ -47,7 +47,7 @@ class HcalTextCalibrations : public edm::ESProducer,
 {
 public:
   HcalTextCalibrations (const edm::ParameterSet& );
-  ~HcalTextCalibrations ();
+  virtual ~HcalTextCalibrations () {};
```

Comment:
If this was already declared before, it means there must already exist an implementation of the destructor. Therefore the chang needed is `virtual ~HcalTextCalibrations();`